imports/ui/stylesheets/base/settings.less

```diff
@@ -35,8 +35,8 @@
 
 @screen-tiny-height: 200px;
 @screen-tiny-width: 400px;
-@screen-mini-height: 320px;
-@screen-mini-width: 570px;
+@screen-mini-height: 280px;
```

Comment:
Nice, I had just noticed that our embedded height was bigger than the height twitter provides for us. Thanks for taking care of this 👍

Reply:
There's some queries for our sizes too, @bent0b0x :
https://github.com/Paratii-Video/paratii-player/blob/dev/imports/ui/stylesheets/base/mixins.less

Reply:
Great, these are awesome!